interface/wx/gdicmn.h

```diff
@@ -903,6 +903,11 @@ class wxColourDatabase
         colour is not found in the database.
     */
     wxString FindName(const wxColour& colour) const;
+
+    /**
+        List all known colours by name.
+    */
```

Comment:
```suggestion
        @since 3.3.0
    */
```

(unless you plan to backport it to 3.2.1, in which case that version should be used).

Reply:
Done. We can change it if/when we backport it.

Reply:
You might want to grab a copy of the proposed sample as it illustrates the other recent issue #22721 of buttons with light colours appearing washed out when hoving the mouse over them. I factored the DrawSystemColours method to use the same setup and loop to draw the DB colours.